app/bundles/LeadBundle/Tests/EventListener/ReportNormalizeSubscriberTest.php

```diff
@@ -22,8 +22,8 @@ class ReportNormalizeSubscriberTest extends MauticMysqlTestCase
      */
     public function testOnReportDisplay(string $value, string $type, array $properties, string $expected): void
     {
-        $fieldModel = self::$container->get('mautic.lead.model.field');
-        \assert($fieldModel instanceof FieldModel);
+        $fieldModel = self::getContainer()->get('mautic.lead.model.field');
+        \assert($fieldModel instanceof FieldModel); // @phpstan-ignore-line
```

Comment:
Why do we have a phpstan ignore here. If this is not directly fixable, i would add it to the baseline. Also not sure what phpstan should not like here.

Reply:
The @phpstan-ignore-line was there, because phpstan complained that the `assert` is useless. It's actually is :)